client/src/main/java/com/alibaba/nacos/client/config/NacosConfigService.java

```diff
@@ -230,6 +233,10 @@ private boolean publishConfigInner(String tenant, String dataId, String group, S
         params.add(group);
         params.add("content");
         params.add(content);
+        //if type is empty, set a default value: json
```

Comment:
Why is the json type the default value, I think text the default value

Reply:
text is also ok, I think json format is more readable in most scenarios

Reply:
I want to be able to follow the original design, the client and the service side of the unity

Reply:
Okay, I will add another commit